### PR TITLE
Update Moonlight

### PR DESCRIPTION
**Pull request by mc-anton-karpov**
## Context

MR contains small improvements for Moonlight and a few useful extensions for Combine.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation including README.md
- [x] My changes generate no new warnings
- [x] My changes generate no breaking changes

### CODE REVIEW

**Review by mc-evgeny-zborovsky — APPROVED** [review]

File: Sources/MoonFoundation/Publisher+Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+public extension Publisher {
+    func sinkValue(_ receiveValue: @escaping ((Self.Output) -> Void)) -> AnyCancellable {
+        sink(
+            receiveCompletion: { _ in },
+            receiveValue: receiveValue
+        )
+    }
+
+    func mapTo() -> Publishers.Map<Self, Void> {
```
Comment (mc-aleksander-stos) [inline Sources/MoonFoundation/Publisher+Extensions.swift:11]: Would it be better to name this method mapToVoid()?

**Review by mc-aleksander-stos — APPROVED** [review]
One suggestion about `mapTo()` function to rename it to `mapToVoid()`

**Review by kuldar-daniel — APPROVED** [review]

File: Sources/MoonFoundation/Publisher+Extensions.swift (hunk 1 above)
Comment (mc-anton-karpov) [reply Sources/MoonFoundation/Publisher+Extensions.swift:11]: not sure if it will be better, let's proceed with just `mapTo` for now

Comment (mc-evgeny-zborovsky) [reply Sources/MoonFoundation/Publisher+Extensions.swift:11]: For the reference `void()` would also work.

**Pull request merged**